index.js

```diff
@@ -210,6 +209,10 @@ module.exports = (function() {
 
 module.exports.credential_cache = credential_cache;
 
+module.exports.set_credential_cache = function(cache){
```

Comment:
Is this function necessary?  I would think the fact that we export `credential_cache` would mean you could do `cimpress_client_request.credential_cache = {thingy}`